content/synthetics/browser_check.md

```diff
@@ -0,0 +1,75 @@
+
+## Overview
+
+Browser Checks are scenarios executed by Datadog on your web applications. They run at configurable periodic intervals from multiple locations around the world, and from multiple devices. These checks verify both that your applications are up and responding to requests, and that any conditions defined in your scenarios are met.
+
+## Configuration
+
+### Test details
+
+Define the configuration of your Browser Check.
+
+{{< img src="synthetics/browser_check/browser_check_configuration.png" alt="Browser Check make request" responsive="true" style="width:80%;">}}
+
+1. Give your Browser Check a name. 
+2. Enter the Start URL. This is the URL from which your Browser Check starts the scenario.
+3. Define on which device to run your check. Available devices are: `Laptop Large`, `Tablet`, and `Mobile Small`.
+4. Pick-up locations to run the test from. Available locations are:
+    * Frankfurt (Request made from an AWS Datacenter)
+    * Tokyo (Request made from an AWS Datacenter)
+5. Choose a Check frequency between "1 run per 5 minute interval" to "1 run per week":
+
+    {{< img src="synthetics/browser_check/check_frequency.png" alt="Check frequency" responsive="true" style="width:80%;">}}
+
+### Notifications
+
+A notification is sent if at least one step of the Browser Check scenario fails. To configure your notifications:
+
+{{< img src="synthetics/browser_check/browser_check_notification.png" alt="Browser check notification" responsive="true" style="width:80%;">}}
+
+1. Select users and/or [services][1] to send the notifications to.
+2. Enter a **message** for the Browser Check. This field allows standard [Markdown formatting][2]. Notification messages include the **message** defined in this section and information about which assertion failed and why.
+3. Click **Save Details and Record Test** to save your Browser Check.
+4. Start to record your test.
+
+## Record test
+
+**Tests can be only recorded from [Google Chrome][3]. To record your test, download the [Datadog Record Test extension for Google Chrome][4].**
+
+
+{{< img src="synthetics/browser_check/browser_check_record_test.png" alt="Browser check record test" responsive="true" >}}
+
+1. Optionally, select **Open in a pop-up** at the upper right of the page to open your test recording in a separate pop-up window in order to avoid sizing issues in the displayed window within Datadog's interface.
+2. Click on **Start recording** to begin recording your Browser Check.
+3. Your actions are recorded and used to create steps within your Browser Check Scenario.
+4. Optionally, use the actions available in the upper left corner to enrich your scenario:
+    {{< img src="synthetics/browser_check/browser_check_assertions.png" alt="Browser Check assertions" responsive="true" style="width:80%;">}}
+
+5. Once you have finished your Scenario, click on **Save and Launch Test**.
+
```

Comment:
Should we add a note on how it's relevant and recommended to finish the scenario by an assertion maybe? @GJSafar

Reply:
Yes, it's very important: if one of the keys identifying the elements is broker while the others are not, the broken key is repaired if and only if the check completely succeeds. Hence, it's very important to make sure the success condition is very clear, to avoid having a "drift" of keys.

No need to explain it that way, but, we have to explain the check should succeed based on the same kind of assertions a human being should be asking. "Does my journey end-up on the right page? --> how do I know it's the right page? --> what text/value indicates me I'm on the right page? --> I create an assertion to validate that text/value are displayed as expected on the page"

Reply:
Second that.